ui/components/ui/chip/chip-with-input.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Chip from '.';
+import PropTypes from 'prop-types';
+import { COLORS } from '../../../helpers/constants/design-system';
+
+export function ChipWithInput({
+  className,
+  borderColor = COLORS.UI1,
+  inputValue,
+  setInputValue,
+}) {
+  return (
+    <Chip className={className} borderColor={borderColor}>
```

Comment:
[req] lets add 'chip--with-input' to className, just in case we need to modify any styles in the future from the main component.